ecosystem/typescript/sdk/examples/typescript/multisig_account.ts

```diff
@@ -40,6 +40,7 @@ const { AccountAddress, EntryFunction, MultiSig, MultiSigTransactionPayload, Tra
     owner1.address(),
     "0x1::multisig_account::OwnedMultisigAccounts",
   );
+  console.log(ownerdMultisigAccounts);
```

Comment:
nice

Reply:
Whoops I was showing Junkil how to write typescript examples and forgot to undo